sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -157,11 +157,12 @@ export default defineComponent({
         applicationProgressDetails.value.appealStatus ===
         StudentAppealStatus.Pending
       ) {
-        // Application is complete but has warnings.
+        // Application is completed but has warnings.
         statusIconDetails.value = STATUS_ICON_WARNING;
```

Comment:
Shouldn't it be complete?

Reply:
Why?

Reply:
"Application was completed" vs "Application is complete".

Reply:
Since it is used in the same way as other places in the code, its fine.

Reply:
If it is used wrongly we can stop using it in this way. I am still looking forward to an indication of which grammar rule we are hurting here 😉